.github/workflows/weekly-cluster.yml

```
@@ -0,0 +1,77 @@
+name: Nightly Test run
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '0 0 * * 1'
```

```suggestion
    - cron:  '0 0 * * 7'
```
Idea is to have it run on a day when the nightly will not run

---

done